### DISCUSSION TITLE
Version 0.15.0 has been released!

### DISCUSSION THREAD

**Discussion by anuraaga** [open-telemetry/opentelemetry-java#2604] [Q&A]
This is our RC release in preparation for 1.0.0. It's been a long journey but we're finally almost there - thanks for bearing with us.

https://github.com/open-telemetry/opentelemetry-java/releases/tag/v0.15.0

**Comment by jkwatson** [comment]
So... as soon as we said `RC`, @bogdandrutu started doing a much-needed scrub through our APIs and SDKs and uncovered a few warts that we're going to address before 1.0. We'll try to iterate quickly, but we'll definitely have another RC version before 1.0, FYI
